src/server/src/test/resources/generate-certs.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
Question: looks like we commit in encryption materials above, but generate them here. Any reason for having both approaches?

Reply:
This is to regenerate the provided encryption materials.

Reply:
Right, why do we want to regenerate them? This is probably fine, just making sure I understand what you're doing.

Reply:
Hopefully we wouldn't (or we could modify it later to regenerate on every CI run), but I like to have generators of stuff we use in the repository, in case someone would for example run into an issue on their refactoring - and wonder why something doesn't work. Having the script gives a clear indication how the certs were generated and be easier to debug - instead of having bunch of encrypted things that were done "somehow". In the same way, if this works, but integration from somewhere else doesn't - this could give a clue.